lib/task_listener.js

```diff
@@ -328,6 +335,59 @@ export default class TaskListener extends EventEmitter {
     if (this.isIdle()) this.emit('idle', this);
   }
 
+  reportCapacityState() {
+    let state = {
+      time: new Date(),
+      idle: this.lastKnownCapacity,
+      busy: this.runningTasks.length,
+    };
+    let time = (state.time.getTime() - this.lastCapacityState.getTime()) / 1000;
+    if (this.lastCapacityState.busy === 0) {
+      this.capacityMonitor.count('running-eq-0', time);
+    }
+    if (this.lastCapacityState.busy >= 1) {
+      this.capacityMonitor.count('running-ge-1', time);
+    }
+    if (this.lastCapacityState.busy >= 2) {
+      this.capacityMonitor.count('running-ge-2', time);
+    }
+    if (this.lastCapacityState.busy >= 3) {
+      this.capacityMonitor.count('running-ge-3', time);
+    }
+    if (this.lastCapacityState.busy >= 4) {
+      this.capacityMonitor.count('running-ge-4', time);
+    }
+    if (this.lastCapacityState.busy >= 6) {
+      this.capacityMonitor.count('running-ge-6', time);
+    }
+    if (this.lastCapacityState.busy >= 8) {
```

Comment:
Why the arbitrary limit of 8 here? Also, do we not care about when it's >= 5 or >= 7? :) Same goes for below this.